process/incubation.md

```diff
@@ -126,9 +131,15 @@ Changes to the Swift Server Ecosystem index page will be announced by the SSWG u
 * Include list of adopters for at least the primary repo ideally laid out in an ADOPTERS.md files or logos on the project website
 * Optionally, have a [Developer Certificate of Origin](https://developercertificate.org) or a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement)
 
-## Process Diagram
+## Security Best Practices
 
-![process diagram](incubation.png)
+Project authors that discover, or been advised of vulnerabilities in their projects, must report them to the SSWG within 10d. Graduated projects are expected to address vulnerabilities within 30d. Reports should be sent to the SSWG using [Swift forums](https://forums.swift.org/c/server/security-updates).
+
+Failure to report or address vulnerabilities may result in the SSWG publishing a security advisory, and could lead to retracting the project's status and listing it under the non-recommended projects list. In some cases, the SSWG may choose to find a technical contributor that can help resolve the security issues to minimize the impsct on the ecosystem. SSWG actions will be decided on a case by case basis and requuire a supermajority vote.
```

Comment:
@ianpartridge @tanner0101 @loganwright @weissi plz provide feedback for this new section per our last sswg meeting